jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/genericfilter/GenericFilterSupport.java

```diff
@@ -200,7 +201,8 @@ public Map<Configuration, Boolean> getConfigurationsMap(GenericFilter filter) {
 
     protected GenericFilterAction<?> createFilterAction(Class<? extends GenericFilterAction<?>> filterActionClass,
                                                         GenericFilter filter) {
-        GenericFilterAction<?> filterAction = actions.create(filterActionClass);
+        String id = filterActionClass.getAnnotation(ActionType.class).value();
```

Comment:
Interesting moment...
Is it ok?

Reply:
Nope, refactor filter code, so `getDefaultFilterActionClasses` > `getDefaultFilterActionIds`